contracts/strategies/LPStrategyIndex.ts

```diff
@@ -0,0 +1,69 @@
+import { ethers } from "ethers";
```

Comment:
is this the script that will be used by OZ to trigger rebalances?

Reply:
Yes, this will be the script in ts. Will need to convert to js before placing it into defender.